.github/workflows/unit_tests.yml

```diff
@@ -59,11 +58,10 @@ jobs:
           TO_TEST="test_no_passport.py or test_datetime.py or test_defaults.py or test_jobqueue.py or test_applicationbuilder.py or test_ratelimiter.py or test_updater.py or test_callbackdatacache.py or test_request.py"
           pytest -v --cov -k "${TO_TEST}" --junit-xml=.test_report_no_optionals_junit.xml
           opt_dep_status=$?
-
+          
+          pip install .[all]
```

Comment:
can you please move this below the line "test the rest"? During review I thought `pip install .[all]` for some reason should now belong to the "test without dependencies" part, which was confusing